pandas/core/generic.py

```diff
@@ -32,6 +31,7 @@
 from pandas._config import config
 
 from pandas._libs import lib
+import json
```

Comment:
the imports are in the wrong order, if you run
```
pre-commit run black --files pandas/core/generic.py
pre-commit run isort --files pandas/core/generic.py
pre-commit run flake8 --files pandas/core/generic.py
```
before committing it should fix up most of the reported issues